### PR TITLE
chore: upgrade @silvermine/standardization to 2.0

### PR DESCRIPTION
**Pull request by pbredenberg**
This commit upgrades Silvermine's standardization library, and removes standards linting
and checking that was formerly run through Grunt.js. Now, linting can be executed using
`npm run standards` instead of `grunt standards`.

Also temporarily disables travis CI for latest Node. See: https://github.com/silvermine/standardization/issues/39

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -110,17 +109,6 @@ module.exports = function(grunt) {
          dist: config.dist.base,
       },
 
-      eslint: {
-         target: config.js.all,
-      },
-
-      stylelint: {
```
Comment (yokuze) [inline Gruntfile.js:117]: `stylelint` was removed here, but I don't see a replacement in the npm scripts.

**Review by yokuze — COMMENTED** [review]
Looks good overall. Just one comment, and there's now an `.idea` folder. Could you update the `.gitignore` in this project to included `.idea`?

**Comment by pbredenberg** [comment]
@yokuze btw - I'm using webstorm this week.

All things done!

**Comment by pbredenberg** [comment]
Closing to start a new one

**Pull request closed without merging**